### PR TITLE
add JDK 12 info to JDK compat page

### PR DESCRIPTION
**Pull request by SethTisue**
updates https://docs.scala-lang.org/overviews/jdk-compatibility/overview.html

### CODE REVIEW

File: _overviews/jdk-compatibility/overview.md [hunk 1]
```diff
@@ -17,13 +17,12 @@ This table shows the first Scala release in each series that works with each JVM
 
 | JVM version | Minimum Scala versions                                                                        |
 |:-----------:|:----------------------------------------------------------------------------------------------|
-| 11          | 2.12.4, 2.11.12, 2.10.7                                                                       |
+| 12          | see [below](#jdk-12-compatibility-notes)                                                      |
+| 11          | 2.12.4, 2.11.12, 2.10.7 (but also, see [below](#jdk-11-compatibility-notes))                  |
```
Comment (dwijnand) [inline _overviews/jdk-compatibility/overview.md]: Relabeled to "JDK 12 compatibility notes"

Comment (dwijnand) [reply _overviews/jdk-compatibility/overview.md]: Ah, nvm both exist. MB

**Comment by SethTisue** [comment]
> You've added specific details that will have to be updated

I've now set a monthly reminder in OmniFocus to check it and update if needed.

**Comment by SethTisue** [comment]
thank you @xuwei-k for your early attention to this

**Comment by SethTisue** [comment]
followup: https://github.com/scala/docs.scala-lang/pull/1315

**Pull request merged**